duckduckgo/Android
# ADS: Site Permissions section migration

Task/Issue URL: https://app.asana.com/0/0/1203411647299660/f

### Description
Migrate Site permissions section to new components

### Steps to test this PR

- Install from this branch
- Go to Settings > Site Permissions

### UI changes
| Before  | After |
| ------ | ----- |
![Screenshot_20221202-114739_DuckDuckGo](https://user-images.githubusercontent.com/20798495/205287199-ca0c20df-0df7-41f5-aac0-3d617553f16c.jpg)|![Screenshot_20221201-142533_DuckDuckGo](https://user-images.githubusercontent.com/20798495/205286967-24cc6b46-9d94-4324-9556-c179803167eb.jpg)|
![Screenshot_20221202-114756_DuckDuckGo](https://user-images.githubusercontent.com/20798495/205287213-f2fdb15d-2750-42f0-9def-7897eedb2fdc.jpg)|![Screenshot_20221201-142543_DuckDuckGo](https://user-images.githubusercontent.com/20798495/205286917-a6a9160c-1bb7-4e48-8255-22a2458b2a31.jpg)|
